<launch>
	<node ns="agent_1" pkg="multi_planner" type="agent_node" name="agent" output="screen" >
		<param name="startX" type="double" value="0.0" />
		<param name="startY" type="double" value="0.0" />
		<param name="startT" type="double" value="0.0" />
		<remap from="/agent_1/agent/agent_feedback" to="/agent_feedback" />
	</node>

	<node ns="agent_2" pkg="multi_planner" type="agent_node" name="agent" output="screen" >
		<param name="startX" type="double" value="5.0" />
		<param name="startY" type="double" value="5.0" />
		<param name="startT" type="double" value="0.0" />
		<remap from="/agent_2/agent/agent_feedback" to="/agent_feedback" />
	</node>



</launch>
